alpine-golang-librdkafka/alpine3.11-golang1.14.1-librdkafka1.4.0-static/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM golang:1.14.1-alpine3.11
```

Comment:
The previous time the 3.11 with the static libraries that the librdkakfa need didn't work. Is now okay?

Reply:
I don't know, I haven't tested it yet. I opened this PR and invited you guys to slowly start considering upgrading Go version. But also feel free to stay with 1.13 for a while more. I'll start testing with these images to see how it goes.